src/Jupyter/ConfigurationSource/IConfigurationSource.cs

```diff
@@ -246,6 +246,14 @@ public T GetOptionOrDefault<T>(string optionName, T defaultValue, Func<string, T
         public string WorkspaceName =>
             GetOptionOrDefault("azure.quantum.workspace.name", string.Empty);
 
+        /// <summary>
+        ///     Whether the resource estimation table should be shown in summary mode or not.
+        ///     In summary mode the explanations are shown as a tooltip, otherwise, they are inlined
+        ///     into the table cells.  The default value is `false`.
+        /// </summary>
+        public bool EstimatorSummary =>
+            GetOptionOrDefault("estimator.summary", false);
```

Comment:
I would name this `estimator.showSummary`, since it's a bool flag.

Reply:
Changed that.